entries/m/mailerlite.com.json

```
@@ -0,0 +1,12 @@
+{
+  "MailerLite": {
+    "domain": "mailerlite.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.mailerlite.com/help/how-to-turn-on-two-factor-authentication-2fa",
+    "categories": [
+      "email"
```

```suggestion
      "marketing"
```

Marketing is probably better.